src/data/valid/Database-interleaved.yaml

```diff
@@ -3932,13 +3945,13 @@ metagenome_sequencing_set:
     part_of:
       - nmdc:wfch-99-dl96
     started_at_time: '2023-08-06T10:00:00Z'
-  - id: nmdc:wfmsa-99-qwertyuiop
+  - id: nmdc:wfmsa-99-qwertyuiop.1
     type: nmdc:MetagenomeSequencing
     name: Sequencing Activity for nmdc:mga0vx38
     has_input:
-      - nmdc:unvalidated_placeholder
+      - nmdc:procsm-99-abcdef
     has_output:
-      - nmdc:22afa3d49b73eaec2e9787a6b88fbdc3
+      - generic:xxx
```

Comment:
@turbomam  generic:xxx should not be allowed. has_output should have a structured pattern for a data object valu ehere.